ui/src/containers/Layout.js

```diff
@@ -99,7 +99,6 @@ class Layout extends Component {
             <PrivateRoute exact path="/about" component={Welcome} />
             <PrivateRoute exact path="/" component={NodeList} />
           </Switch>
-          )}
```

Comment:
unrelated change, shouldn't be in this commit

Reply:
I rename the commit